### PR TITLE
Add gateway component to component status

### LINKED ISSUE #217: Add tempo gateway to component status

### PR DESCRIPTION
**Pull request by rubenvp8510**
Resolve #217

### CODE REVIEW

File: bundle/manifests/tempo-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -481,7 +487,7 @@ spec:
                 - --config=controller_manager_config.yaml
                 command:
                 - /manager
-                image: ghcr.io/os-observability/tempo-operator/tempo-operator:v0.0.1
+                image: ghcr.io/rvargasp/tempo-operator/tempo-operator:v0.0.1
```
Comment (iblancasa) [inline bundle/manifests/tempo-operator.clusterserviceversion.yaml:490]: Revert this

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: ghcr.io/rvargasp/tempo-operator/tempo-operator
```
Comment (iblancasa) [inline config/manager/kustomization.yaml:15]: Revert this

**Review by andreasgerstmayr — APPROVED** [review]

**Review by iblancasa — APPROVED** [review]

**Comment by rubenvp8510** [comment]
Don't merge, need to test after the rebase.

**Comment by frzifus** [comment]
rebased

**Review by frzifus — APPROVED** [review]

**Pull request merged**